check_raid.pl

```diff
@@ -559,15 +558,41 @@ sub program_names {
 
 sub commands {
 	{
-		'status' => ['-|', '@CMD'],
+		'status' => [>&2', '@CMD'],
```

Comment:
Your right, the apostrophe is missing and clearly shouldn't be. When using it correctly however, it does see both stderr and stdout, so if if SVM isn't in use, we get the message to stderr and disable the plug-in. If it returns data to stdout, this is processed too and the disks and status are returned.

This was what I was expecting based on the comment in the code

Regards

Matt

________________________________
From: Elan Ruusamäe [mailto:notifications@github.com]
Sent: 16 February 2015 13:43
To: glensc/nagios-plugin-check_raid
Cc: Noble, Matthew
Subject: Re: [nagios-plugin-check_raid] Enable sds support (#96)


In check_raid.pl<https://github.com/glensc/nagios-plugin-check_raid/pull/96#discussion_r24750990>:

> @@ -559,15 +558,41 @@ sub program_names {
>
>  sub commands {
>       {
> -             'status' => ['-|', '@CMD'],
> +             'status' => [>&2', '@CMD'],


On 16.02.2015 15:37, noblemtw wrote: I think '>&2' is actually correct, it's a special op used by the command function,
also, >&2 special op will switch to using STDERR instead of STDOUT, i.e it will not merge two streams, but switch to stderr. and by your comments you are suggesting it really redirects and as result check_raid sees both streams merged?
...
-- glen

-
Reply to this email directly or view it on GitHub<https://github.com/glensc/nagios-plugin-check_raid/pull/96/files#r24750990>.

--------------------------------------------------------------------
Save Paper - Do you really need to print this e-mail?

Visit www.virginmedia.com for more information, and more fun.

This email and any attachments are or may be confidential and legally privileged
and are sent solely for the attention of the addressee(s). If you have received this
email in error, please delete it from your system: its use, disclosure or copying is
unauthorised. Statements and opinions expressed in this email may not represent
those of Virgin Media. Any representations or commitments in this email are
subject to contract. 

Registered office: Media House, Bartley Wood Business Park, Hook, Hampshire, RG27 9UP
Registered in England and Wales with number 2591237